network/p2p/libp2pNode.go

```diff
@@ -19,57 +19,24 @@ import (
 	"github.com/onflow/flow-go/network/p2p/unicast/protocols"
 )
 
-// LibP2PNode represents a flow libp2p node. It provides the network layer with the necessary interface to
-// control the underlying libp2p node. It is essentially the flow wrapper around the libp2p node, and allows
-// us to define different types of libp2p nodes that can operate in different ways by overriding these methods.
-// TODO: this interface is highly coupled with the current implementation of the libp2p node. We should
-//
-//	consider refactoring it to be more generic and less coupled with the current implementation.
-//	https://github.com/dapperlabs/flow-go/issues/6575
-type LibP2PNode interface {
-	module.ReadyDoneAware
-	Subscriptions
-	// PeerConnections connection status information per peer.
-	PeerConnections
-	// PeerScore exposes the peer score API.
-	PeerScore
-	// DisallowListNotificationConsumer exposes the disallow list notification consumer API for the node so that
-	// it will be notified when a new disallow list update is distributed.
-	DisallowListNotificationConsumer
-	// CollectionClusterChangesConsumer  is the interface for consuming the events of changes in the collection cluster.
-	// This is used to notify the node of changes in the collection cluster.
-	// LibP2PNode implements this interface and consumes the events to be notified of changes in the clustering channels.
-	// The clustering channels are used by the collection nodes of a cluster to communicate with each other.
-	// As the cluster (and hence their cluster channels) of collection nodes changes over time (per epoch) the node needs to be notified of these changes.
-	CollectionClusterChangesConsumer
-	// DisallowListOracle exposes the disallow list oracle API for external consumers to query about the disallow list.
-	DisallowListOracle
+// Node single node management capabilities
+type Node interface {
```

Comment:
```suggestion
type LibP2PNode interface {
```
Otherwise will be confused with other definitions of node in our codebase.

Reply:
https://github.com/onflow/flow-go/pull/4630/commits/455147a08f62bace4ff3fb18c2cc6ae018d3ca68